demo/app.config.ts

```diff
@@ -7,9 +7,8 @@ export default {
   },
   assetBundlePatterns: ['**/*'],
   extra: {
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
     baseUrl: process.env.BASE_URL || 'http://0.0.0.0:8085',
+    googleMapsApiKey: process.env.GOOGLE_MAPS_API_KEY || '', // Leaving the key empty will enable dev-only experience
```

Comment:
What is the difference with dev-only mode? Is there a watermark, or some missing functionality?

Reply:
yes, there's a water mark and a popup when UI loads, and the markers are not rendering in this situation.